packages/grid/src/vaadin-grid-selection-column-base-mixin.js

```diff
@@ -107,6 +107,12 @@ export const GridSelectionColumnBaseMixin = (superClass) =>
         root.appendChild(checkbox);
         // Add listener after appending, so we can skip the initial change event
         checkbox.addEventListener('checked-changed', this.__onSelectAllCheckedChanged.bind(this));
+        // Toggle on Space without having to enter interaction mode first
+        column._headerCell.addEventListener('keydown', (e) => {
```

Comment:
Might make sense to remove this listener when the component gets disconnected.

Reply:
Do you think it's needed? Other listeners added here are not being removed.

Reply:
Refactored to use `.bind(this)` as for other listeners. This should be enough.